Xamarin.Forms.Platform.UAP/DatePickerRenderer.cs

```diff
@@ -65,6 +66,16 @@ void ControlOnLoaded(object sender, RoutedEventArgs routedEventArgs)
 			UpdateTextColor();
 		}
 
+		internal override void OnElementFocusChangeRequested(object sender, VisualElement.FocusRequestArgs args)
+		{
+			base.OnElementFocusChangeRequested(sender, args);
+
+			// Show a picker fly out on focus to match iOS and Android behavior
+			var flyout = new DatePickerFlyout { Placement = FlyoutPlacementMode.Bottom, Date = Control.Date };
+			flyout.DatePicked += (p, e) => Control.Date = p.Date;
+			flyout.ShowAt(Control);
```

Comment:
How does this behave when the picker is set to not visible? See for instance #5159

Reply:
I did a quick test on when it's not visible, it opens but I'm not sure the positioning is great.

![image](https://user-images.githubusercontent.com/15127788/67102424-08163f00-f189-11e9-86ba-6c082d55bc1b.png)

What I'm thinking to do is if the control isn't visible use the Full placement of the flyout, which will have the picker popup fill the current screen.

![image](https://user-images.githubusercontent.com/15127788/67102673-7b1fb580-f189-11e9-912f-2c6b0c735559.png)

Thoughts?

I'll also update the issue page to include both a visible and not visible control.